docs/product/issues/suspect-commits/index.mdx

```diff
@@ -43,11 +43,11 @@ Suspect commits can be enabled using the following integrations:
 In [sentry.io](https://sentry.io):
 
 1. Go to **Settings > Integrations**.
-1. Click the integration you’d like to use.
-1. Configure the integration if you haven’t already.
-1. Add the repository you'd like to use.
+2. Click the integration you’d like to use.
+3. Configure the integration if you haven’t already.
+4. Add the repository you'd like to use.
```

Comment:
Not a big deal, but I would err on the side of not changing these and keeping them as 1. so it's easier to update and add/remove steps in the future if needed.

Reply:
ahh, i had no idea why they were all `1.` but I saw it didn't matter when rendered. TY!